src/python/dxpy/sugar/transfers.py

```diff
@@ -64,7 +64,8 @@
 
 
 LOG = logging.getLogger()
-SPECIAL_RE = re.compile(r"[^\w.]")
+SPECIAL_RE = re.compile(r"[^\w-.]")
```

Comment:
```suggestion
SPECIAL_RE = re.compile(r"[^\w\-\.]")
```

Only the `\-` is needed to not misinterpret the RE and a range [\w ... .], which is invalid and shouldn't compile. Added the `\.` just to future proof a bit if someone adds characters here.

Reply:
In an r-string, the escapes are redundant